Makefile

```diff
@@ -203,6 +203,12 @@ codegen:
 
 ### Docker images ###
 
+.PHONY: base-ubuntu
+base-ubuntu:
```

Comment:
this target is never used, you also use the OVS VERSION to tag the image not the antrea version. So you should remove this target

Reply:
Removed.